source/includes/steps-development-mode-realm-schema.yaml

```
@@ -0,0 +1,91 @@
+title: Enable ``Development Mode``
+ref: enable-development-mode
+content: |
+
+  You can alter or create a Realm Object Model through your mobile
+  client SDK. Changes to your Realm Object Model will only be allowed
+  when :guilabel:`Development Mode` is on in the MongoDB Realm UI.
+  MongoDB Realm will reflect these changes to your Realm Object Model in
+  your Realm Schema used for Atlas. To enable :guilabel:`Development
+  Mode`, navigate to the ``SDKs`` page, the "Development Mode is ON"
+  modal will appear immediately, indicating that Realm Object Modal
```

```suggestion
  modal appears immediately, indicating that Realm Object Model
```

- Avoid future tense. Use active, present tense instead.
- Can we indicate that this only happens the first time you visit the SDKs page?
- How do I turn this back on if it's not my first time, i.e. it doesn't automatically turn on